permission_handler_platform_interface/lib/src/permission_status.dart

```diff
@@ -18,10 +19,14 @@ enum PermissionStatus {
   /// *Only supported on iOS (iOS14+).*
   limited,
 
+  /// *On Android:*
   /// Permission to the requested feature is permanently denied, the permission
   /// dialog will not be shown when requesting this permission. The user may
   /// still change the permission status in the settings.
-  /// *Only supported on Android.*
+  ///
+  /// *On iOs:*
+  /// Permission to the requested feature has been denied. The user may still
+  /// change the permission status in the settings.
```

Comment:
The documentation could be the same for Android and iOS (behaviour is the same):

```dart
  /// Permission to the requested feature is permanently denied, the permission
  /// dialog will not be shown when requesting this permission. The user may
  /// still change the permission status in the settings.
```